src/main/java/billgates/usecases/bill_import/BillImportRequestModel.java

```diff
@@ -0,0 +1,28 @@
+package billgates.usecases.bill_import;
+
+import billgates.database.QueryBillData;
+import billgates.database.QueryEntryData;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Objects;
+
+public class BillImportRequestModel {
+    private List<List<String>> csvFile;
+
+    public BillImportRequestModel(List<List<String>> csvFile) {
```

Comment:
Is this input a CSV File? Do we want to use any libraries like OpenCSV or some scanner to read the CSV file? 

@ScottCTD